docs/DataManagementAnalytics-Prerequisites.md

```diff
@@ -48,22 +48,27 @@ To help you more quickly understand the structure of the repository, here is an
 For now, we are recommending to select one of the regions mentioned below. The list of regions is limited for now due to the fact that not all services and features are available in all regions. This is mostly related to the fact that we are recommending to leverage at least the zone-redundant storage replication option for all your central Data Lakes in the Data Landing Zones. Since zone-redundant storage is not available in all regions, we are limiting the regions in the Deploy to Azure experience. If you are planning to deploy the Data Management Zone and Data Landing Zone to a region that is not listed below, then please change the setting in the corresponding bicep files in this repository. Officially supported regions are:
```

Comment:
Should we change "officially supported" ? I mean we do not want to restrict adoption of DMA to regions where let's say AZs are available. As a parttern, DMA is composed of services which are available out of most regions. Using word such as "Support" could be misinterpreted as MSFT's support for various services. 

Perhaps, "DMA deployment has been tested in the following regions:"

Reply:
Thanks for the suggestion. Updated the wording.